src/content/configuration/externals.md

```diff
@@ -144,6 +145,23 @@ module.exports = {
 
 This syntax is used to describe all the possible ways that an external library can be made available. `lodash` here is available as `lodash` under AMD and CommonJS module systems but available as `_` in a global variable form. `subtract` here is available via the property `subtract` under the global `math` object (e.g. `window['math']['subtract']`).
 
+### `byLayer`
```

Comment:
Let's move this section to the end. All others are talking about types of `externals`, it's sort of abrupt to insert `byLayer` here.

Reply:
fixed